core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -345,4 +354,17 @@ private void persistPortInformation(final int httpPort, final Optional<Integer>
       LOG.warn("Error writing ports file", e);
     }
   }
+
+  private String buildCorsRegexFromConfig() {
+    if (config.getCorsAllowedOrigins().isEmpty()) {
+      return "";
+    }
+    if (config.getCorsAllowedOrigins().contains("*")) {
+      return "*";
```

Comment:
with upcoming Vertx4, this needs to be `.*` in order to work with MetaMask etc. (i.e. non-http protocols)

Reply:
I'll wait till Vertx 4 change has been done first then and add this afterwards with `.*` change

Reply:
done